spec/models/note_spec.rb

```diff
@@ -1,17 +1,125 @@
 require 'rails_helper'
 
 RSpec.describe Note, type: :model do
-  subject(:note) { build(:note) }
+  subject(:note) { create(:note, utility: north_utility) }
+
+  let(:north_utility) { create(:north_utility, code: 1) }
 
   %i[title content type].each do |value|
     it { is_expected.to validate_presence_of(value) }
   end
 
-  it { is_expected.to belong_to :user }
-
   it { is_expected.to have_one(:utility).through(:user) }
 
   it 'has a valid factory' do
     expect(subject).to be_valid
   end
+
+  describe '#word_count' do
+    it 'returns the correct number of words' do
+      subject.content = 'it has four words'
+      expect(subject.word_count).to equal 4
+    end
+  end
+
+  describe '#content_length' do
+    context 'when content has less than 50 words' do
+      it 'returns short' do
+        subject.content = 'word ' * 49
+        expect(subject.content_length).to eq 'short'
+      end
+    end
+
+    context 'when content has 50 words' do
+      it 'returns short' do
+        subject.content = 'word ' * 50
+        expect(subject.content_length).to eq 'short'
+      end
+    end
+
+    context 'when content has less than 100 words' do
+      it 'returns medium' do
+        subject.content = 'word ' * 99
+        expect(subject.content_length).to eq 'medium'
+      end
+    end
+
+    context 'when content has 100 words' do
+      it 'returns medium' do
+        subject.content = 'word ' * 100
+        expect(subject.content_length).to eq 'medium'
+      end
+    end
+
+    context 'when content has more than 100 words' do
+      it 'returns long' do
+        subject.content = 'word ' * 101
+        expect(subject.content_length).to eq 'long'
+      end
+    end
+  end
```

Comment:
Acá seria necesario hacer dos grandes contextos para ambas Utilities y probar los casos.
Cuando creas la nota:
`subject(:note) { create(:note, utility: north_utility) }`
podrías ademas, agregar como variable `content`:
`subject(:note) { create(:note, utility: north_utility, content: content) }`
Luego en cada context por ejemplo el siguiente podrías hacer esto:
``` ruby
context 'when content has less than 50 words' do
  let(:content) { 'word ' * 49 }

  it 'returns short' do
    expect(subject.content_length).to eq 'short'
  end
end
```
De esta forma no estarías metiendo contexto adentro de tus `it`

Reply:
Buenísimo, gracias!